baybe/searchspace/core.py

```diff
@@ -310,9 +313,18 @@ def validate_searchspace_from_config(specs: dict, _) -> None:
             constraints = converter.structure(specs["constraints"], List[Constraint])
             validate_constraints(constraints, parameters)
 
-    # For all other types, validate by construction
     else:
-        converter.structure(specs, SearchSpace)
+        discrete_subspace_specs = specs.get("discrete", {})
+        if discrete_subspace_specs.get("constructor", None) == "from_simplex":
+            # Validate discrete simplex subspace
+            _validation_converter = converter.copy()
```

Comment:
Why is this `.copy()` needed here?

Reply:
because in this section we dont want to modify the original converter